### ISSUE TITLE
Aspect ratio

### ISSUE THREAD

**Issue by skrodahl** [anthonycaccese/es-theme-picade#1]
Excellent theme, but the Picade has a 4:3 aspect ratio. Could this be changed easily in the them?

**Comment by anthonycaccese** [comment]
Hi @skrodahl, thank you for checking it out!  The theme is currently designed for 4x3 - possible to send me details of your build so I can help check it out?

**Comment by skrodahl** [comment]
Hi,
I've tried more than fifty themes on my Picade, this is one of my favorites.

The problem is that the graphics wrap over from the right hand side to the left. I just assumed it was because of the aspect ratio.

I've restarted EmulationStation, and I've tried updating the theme (which is up to date, btw).

![IMG_20200310_212057](https://user-images.githubusercontent.com/38037686/76356254-eecb8600-6315-11ea-9d7f-1dee86e7c371.jpg)

(Sorry about the rotated image, I've no idea why Github did that)
